docs/openapi.json

```diff
@@ -1468,6 +1525,79 @@
                     }
                 }
             },
+            "models.AdvisoryMetadata": {
+                "type": "object",
+                "properties": {
+                    "advisoryTypeID": {
+                        "type": "integer"
+                    },
+                    "cveList": {
+                        "type": "string"
+                    },
+                    "description": {
+                        "type": "string"
+                    },
+                    "id": {
+                        "type": "integer"
+                    },
+                    "modifiedDate": {
+                        "type": "string"
+                    },
+                    "name": {
+                        "type": "string"
+                    },
+                    "packageData": {
+                        "type": "string"
+                    },
+                    "publicDate": {
+                        "type": "string"
+                    },
+                    "severityID": {
+                        "type": "integer"
+                    },
+                    "solution": {
+                        "type": "string"
+                    },
+                    "summary": {
+                        "type": "string"
+                    },
+                    "synopsis": {
+                        "type": "string"
+                    },
+                    "url": {
+                        "type": "string"
+                    }
+                }
+            },
+            "models.Package": {
+                "type": "object",
+                "properties": {
+                    "advisoryID": {
+                        "type": "integer"
+                    },
+                    "descriptionHash": {
+                        "type": "array",
+                        "items": {
+                            "type": "integer"
+                        }
+                    },
+                    "evra": {
+                        "type": "string"
+                    },
+                    "id": {
+                        "type": "integer"
+                    },
+                    "nameID": {
+                        "type": "integer"
+                    },
+                    "summaryHash": {
```

Comment:
Hashes are useless for endpoint caller. He needs summary or description itself. Hashes are used only internally for pairing.